src/connection/index.ts

```diff
@@ -107,6 +107,19 @@ export const uniwalletConnectConnection: Connection = {
   isNew: true,
```

Comment:
Will `initializeConnector<UniwalletConnect>` fail to initialize once V1 shuts down? If this throws a synchronous error it will crash the application, I think. (If it is synchronous, I don't think it will matter.)